tests/sentry/feedback/test_feedback_ingest.py

```diff
@@ -0,0 +1,180 @@
+            assert response.status_code == 201
+
+            # Feedback object exists
+            feedback_list = Feedback.objects.all()
+            assert len(feedback_list) == 1
+
+            # Feedback object is what was posted
+            feedback = feedback_list[0]
+            assert feedback.data["type"] == "transaction"
+            assert feedback.data["environment"] == "prod"
+            assert feedback.data["sdk"]["name"] == "sentry.javascript.react"
+            assert feedback.data["transaction_info"]["source"] == "route"
+            assert feedback.data["tags"]["sentry_version"] == "23.9.0.dev0"
+            assert feedback.data["release"] == "frontend@40f88cd929122ac73749cc48f0ddb9aa223449ff"
+            assert feedback.data["user"]["name"] == "Josh Ferge"
+            assert (
+                feedback.data["request"]["url"]
+                == "https://sentry.sentry.io/replays/?project=11276&statsPeriod=7d"
+            )
+            assert feedback.data["platform"] == "javascript"
+            assert feedback.message == "This website is great!"
+
+    def test_no_feature_enabled(self):
+        # Feature disabled should lead to unsuccessful save
+        with self.feature({"organizations:user-feedback-ingest": False}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path, data=test_data, **self.dsn_auth_headers)
+            assert response.status_code == 404
+
+    def test_not_authorized(self):
+        # No authorization should lead to unsuccessful save
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path, data=test_data)
+            assert response.status_code == 401
+            assert response.data == {"detail": "Authentication credentials were not provided."}
+
+    def test_wrong_input(self):
+        # Wrong inputs should lead to failed validation
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path, data=wrong_test_data, **self.dsn_auth_headers)
+            assert response.status_code == 400
+
+    def test_no_event_id(self):
+        # Event ID is required for a successful post
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(
+                path, data=missing_event_id_test_data, **self.dsn_auth_headers
+            )
+            assert response.status_code == 400
+
+    def test_wrong_type(self):
+        # Fields should be correct type
+        with self.feature({"organizations:user-feedback-ingest": True}):
+            path = reverse(self.endpoint)
+            response = self.client.post(path, data=wrong_type_test_data, **self.dsn_auth_headers)
+            assert response.status_code == 400
```

Comment:
One thing we'll want to do is to assert the reason the 400 was raised.  An assertion on `response.content` tells us why it failed.  This is important for catching regressions.  Because if this test actually produces two errors (now or in the future) then a mistaken removal of the "wrong_type" validation will still return a 400 response here even though the invariant has been broken.

Anywhere we're doing a bare `400` status code assertion you should add additional coverage to assert why the request failed.